openhexa/cli/settings.py

```diff
@@ -23,7 +23,7 @@ def _open_config():
         config.read_string(
             """
         [openhexa]
-        url=https://api.openhexa.org
+        url=https://app.openhexa.org
```

Comment:
Did you try with this URL ?

Reply:
It's true that it may work now that we proxy /graphql from the frontend to the backend...